File: src/main/java/gr/apt/repository/LeaveRepository.java
package gr.apt.repository;

import gr.apt.persistence.entity.Leave;
import io.quarkus.hibernate.orm.panache.PanacheRepositoryBase;

import javax.enterprise.context.ApplicationScoped;
import java.math.BigInteger;

@ApplicationScoped
public class LeaveRepository implements PanacheRepositoryBase<Leave, BigInteger> {
}

File: src/main/java/gr/apt/repository/LeaveRepository.kt
package gr.apt.repository

import gr.apt.persistence.entity.Leave
import io.quarkus.hibernate.orm.panache.PanacheRepositoryBase
import java.math.BigInteger
import javax.enterprise.context.ApplicationScoped

@ApplicationScoped
class LeaveRepository : PanacheRepositoryBase<Leave?, BigInteger?>